src/jvmMain/kotlin/io/beatmaps/api/collaboration.kt

```diff
@@ -119,6 +120,8 @@ fun Route.collaborationRoute() {
                             it[accepted] = true
                         }
 
+                        call.pub("beatmaps", "maps.${map.id}.updated.collaborators.added", null, map.id.value)
```

Comment:
Calling a publish from inside a transaction doesn't guarantee the transaction will be committed before the message is processed.

Reply:
You are right! I now pulled it out of the transaction - I hope they way I did is fine. Otherwise, happy to hear suggestions!